### PR TITLE
chore: adding storybook stories to review colors

### PR DESCRIPTION
**Pull request by georgewrmarshall**
## **Description**

This pull request adds Storybook stories for the following components: `AppLoadingSpinner`, `EthOverview`, and `LoadingScreen`. The goal is to provide better visual documentation and enable easier testing of these components within Storybook. It also breaks up a PR to remove all deprecated secondary colors. 

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25083?quickstart=1)
## **Related issues**

Part of: https://github.com/MetaMask/metamask-extension/issues/24964

## **Manual testing steps**

1. Go to storybook build of this PR or start the Storybook server using `yarn storybook`.
2. In the Storybook interface, navigate to the stories for `AppLoadingSpinner`, `EthOverview`, and `LoadingScreen`.
3. Verify that the components render correctly and interact as expected

### **Before**

https://github.com/MetaMask/metamask-extension/assets/8112138/f030a908-b6f9-4877-9e1a-16f10a16a765

### **After**

https://github.com/MetaMask/metamask-extension/assets/8112138/37c4fd40-7e92-4480-ba8b-5d6b0ac8cd13

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability.
- [x] I’ve included tests if applicable.
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and/or screenshots.

### CODE REVIEW

**Comment by brianacnguyen** [comment]
is app loading spinner supposed to be just gray like that?

**Comment by georgewrmarshall** [comment]
> is app loading spinner supposed to be just gray like that?

Good question! This PR just adds the Storybook files and does not touch any component colors. This is what is currently in production. I will create a subsequent PR that resolves any color issues.

**Comment by georgewrmarshall** [comment]
> > is app loading spinner supposed to be just gray like that?
> 
> Good question! This PR just adds the Storybook files and does not touch any component colors. This is what is currently in production. I will create a subsequent PR that resolves any color issues.

**Comment by brianacnguyen** [comment]
> > is app loading spinner supposed to be just gray like that?
> 
> Good question! This PR just adds the Storybook files and does not touch any component colors. This is what is currently in production. This was part of your suggestion to break up this PR [#24971 (comment)](https://github.com/MetaMask/metamask-extension/pull/24971#issuecomment-2151100016)

Hmm can you double check if its indeed broken on the component side or is there anything missing from the story pov? if it's indeed broken then i can just approve it so there's at least a story for someone to come in and fix

**Comment by georgewrmarshall** [comment]
> > > is app loading spinner supposed to be just gray like that?
> > 
> > 
> > Good question! This PR just adds the Storybook files and does not touch any component colors. This is what is currently in production. This was part of your suggestion to break up this PR [#24971 (comment)](https://github.com/MetaMask/metamask-extension/pull/24971#issuecomment-2151100016)
> 
> Hmm can you double check if its indeed broken on the component side or is there anything missing from the story pov? if it's indeed broken then i can just approve it so there's at least a story for someone to come in and fix

Yup, definitely the component, you can check by going to the storybook build of this PR to check the component. It's using secondary-muted which is a transparent orange https://github.com/MetaMask/metamask-extension/blob/491515368dcd9f9203607092b0781e3b85f95281/ui/components/app/app-loading-spinner/app-loading-spinner.js#L22


https://github.com/MetaMask/metamask-extension/assets/8112138/2d05f8f8-02c2-444b-bdae-5a8ec0be1a5a

**Review by brianacnguyen — DISMISSED** [review]

File: ui/components/ui/loading-screen/loading-screen.stories.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export default {
+  title: 'Components/UI/LoadingScreen',
+  component: LoadingScreen,
+};
+
+const Template = (args) => <LoadingScreen {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  // header: <h1>Loading Screen</h1>,
```
Comment (georgewrmarshall) [inline ui/components/ui/loading-screen/loading-screen.stories.js:13]:
```suggestion
```

**Review by brianacnguyen — DISMISSED** [review]

**Review by brianacnguyen — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**